webapp/components/User/User.vue

```diff
@@ -11,17 +11,20 @@
         <div @mouseover="openMenu(true)" @mouseleave="closeMenu(true)">
           <hc-avatar class="avatar" :user="user" />
           <div>
-            <b class="username">{{ userName | truncate(18) }}</b>
-          </div>
-          <!-- Time -->
-          <div v-if="dateTime" style="display: inline;">
-            <ds-text align="left" size="small" color="soft">
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2019-09-18T17:42:27Z" title="Wednesday, September 18th 2019, 7:42:27 pm +02:00">Sep 18, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -14,14 +14,15 @@
             <b class="username">{{ userName | truncate(18) }}</b>
           </div>
           <!-- Time -->
-          <div v-if="dateTime" style="display: inline;">
-            <ds-text align="left" size="small" color="soft">
+          <ds-text align="right" size="small" color="soft">
+            <span class="slug">{{ userSlug }}</span>
+            <div v-if="dateTime" style="display: inline;">
```


If you use a `<span>` here instead of the `<div>` there's no need for the `inline` style. It also makes more sense semantically. ;)